package.json

```diff
@@ -6,24 +6,25 @@
   "scripts": {
     "predev": "yarn codegen",
     "dev": "ts-node-dev --no-notify --respawn --transpile-only src/server",
-    "start": "node dist/server",
+    "start": "./node_modules/.bin/nodemon --optimize_for_size --max_old_space_size=920 --gc_interval=100 dist/server.js",
     "clean": "rm -rf dist",
-    "build": "yarn -s run clean && yarn -s run codegen && tsc",
+    "build": "yarn -s run clean && tsc",
     "codegen": "graphql-codegen",
     "codegen:watch": "graphql-codegen --watch",
+    "heroku-postbuild": "yarn build && npx sequelize-cli db:migrate && cd frontend && yarn install && yarn build:styles && yarn build",
```

Comment:
This is what Heroku runs instead of `yarn build`

Reply:
Suggestion (non-blocking): can we seed the database as well?

Reply:
That could get fairly annoying since it would either duplicate whatever data is already seeded from last time or wipe out anything we've created since for testing purposes which would be pretty annoying if someone (especially a non-technical contributor) were testing something on staging and an auto-deploy wiped their data.

Reply:
You're right 👍